src/Nox.EntityFramework.SqlServer/SqlServerDatabaseProvider.cs

```diff
@@ -34,6 +35,21 @@ protected override IList<IndexBuilder> ConfigureUniqueAttributeConstraints(IEnti
 
         return result;
     }
+    
+    protected override void ConfigureAutoNumberAttributeSequences(ModelBuilder modelBuilder, Entity entity)
+    {
+        if (entity.Attributes is not { Count: > 0 }) return;
+        foreach (var property in entity.Attributes.Where(e=>e.Type == NoxType.AutoNumber))
+        {
+            var typeOptions = property.AutoNumberTypeOptions ?? new AutoNumberTypeOptions();
+                
+            var seqName = $"Seq{entity.Name}{property.Name}";
+                
+            modelBuilder.HasSequence<long>(seqName, entity.Persistence!.Schema)
```

Comment:
We should not set the Persistence Schema when defining the Sequence.

Reply:
Desired changes has been applied